arctic/store/version_store.py

```diff
@@ -517,6 +522,31 @@ def append(self, symbol, data, metadata=None, prune_previous_version=True, upser
         return VersionedItem(symbol=symbol, library=self._arctic_lib.get_name(), version=version['version'],
                              metadata=version.pop('metadata', None), data=None)
 
+    @mongo_retry
+    def update(self, symbol, data, **kwargs):
+        """
+        Find an existing chunk of data and update (overwrite) it. This WILL NOT
```

Comment:
It would be good to know from the docstring to detail what update means in this context. Is it a replacement of all data in a particular date-range chunk, or a merge, or a replacement in the date-range bounds contained within `data`